src/pages/index.md

```diff
@@ -124,47 +125,51 @@ sample.vc
 
 </details>
 
-<br />You will notice that this VC looks much like the one you issued in the
-previous step, but with a "proof" section attached with contains the issuer's
-signature and metadata for verifying it.  Verifying this signature is as simple
-as another single-line DIDKit call:
+<br />
+This VC looks much like the one you issued in the
+previous step, but with a "proof" section attached which contains the issuer's
+signature and metadata for verifying it. 
+
+To verify this signature:
 
 ```bash
 didkit vc-verify-credential -p assertionMethod <example.vc >result.json
 ```
 
-That will spit out a verbose response as a JSON file listing the checks passed, warnings, and errors. If everything is set up properly, you should see this when you `nano result.json`:
+That will spit out a verbose response as a JSON file listing the checks passed, warnings, and errors. If everything is set up properly, you should see this when you `cat result.json`:
 
 ```json
 {"checks":["proof"],"warnings":[],"errors":[]}
 ```
 
 You might be asking yourself, though, how did DIDKit get from `"issuer": "did:web:demo.spruceid.com:2021:vc-faucet"` to having the public key (and knowing the *type* of the public key) to be able to verify the signature in the proof? This is the magic of DIDs!
 
-<details><summary>Detailed explanation of the ✨Magic of Step 1.5✨</summary>
-
-<h3>The [often implicit!] resolution step</h3>
-
+<details><summary>Details of ✨<strong>Magical Step 1.5</strong>✨ - the <em>(often implicit!)</em> resolution step</summary>
```

Comment:
<summary>✨<strong>Magical Step 1.5</strong>✨: the <em>(often implicit!)</em> resolution step</summary>